src/devices/Bmx280/BmxBase.cs

```diff
@@ -88,7 +81,6 @@ public PowerMode ReadPowerMode()
                 return PowerMode.Forced;
             }
         }
-
```

Comment:
NIT: We shouldn't remove this line, as we do want to have one empty line separating each method.